controllers/flp/flp_pipeline_builder.go

```diff
@@ -66,21 +66,29 @@ func (b *PipelineBuilder) AddProcessorStages() error {
 	allLabels = append(allLabels, b.detectedSubnets...)
 	flpLabels := subnetLabelsToFLP(allLabels)
 
+	var srcMacInput, dstMacInput string
```

Comment:
I think we will have an issue with flows on pods identified out of MAC: the app/infra layer label won't be set, because it uses IP address only to search in index and figure out the namespace. cf line 100 this file.
I see 2 options:
- Either we provide MacInputs as well via `K8sInfraRule`
- Or we change the way to get namespace, and instead of doing an informer lookup in FLP like it's currently done [here](https://github.com/netobserv/flowlogs-pipeline/blob/f5d2cefe616774d46ac94eba94a62217ee536b46/pkg/pipeline/transform/kubernetes/enrich.go#L134), we change that to look at the namespace field already enriched in the flow GenericMap.

I tend to prefer option 2, but it means the enriching stage becomes a pre-requisite for the Infra-layer-enrich stage.

@jpinsonneau @OlivierCazade wdyt?

Reply:
I updated https://github.com/netobserv/flowlogs-pipeline/pull/674 with that 2d option.
It's a breaking change so I need to update this PR as well